src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -213,34 +226,89 @@ const useGroupInfo = (channelId = null) => {
-        });
-      }
+  const onRemoveUser = async () => {
+    setOpenModal(false);
+    const responseChannelData = await removeMemberGroup({
+      channelId,
+      targetUserId: selectedUser?.userId
+    });
+
+    try {
+      const {channelName} = getChannelListInfo(
+        responseChannelData.data,
+        signedProfileId,
+        anonProfileId
+      );
+      const channelList = await ChannelListSchema.getSchemaById(localDb, channelId);
+      channelList.name = channelName;
+      await channelList.save(localDb);
+
+      setSelectedChannel(channelList);
+
+      await UserSchema.deleteByUserId(localDb, selectedUser?.userId, channelId);
+    } catch (e) {
+      console.log('error on memberSchema');
+      console.log(JSON.stringify(e));
+      console.log(e);
     }
+
+    refresh('channelList');
+    refresh('chat');
+    refresh('channelInfo');
+
+    setTimeout(() => {
+      navigation.navigate('SignedChatScreen');
+    }, 500);
   };
 
-  const onRemoveUser = async () => {
-    setOpenModal(false);
+  const onAddMember = async (selectedUsers) => {
+    const responseChannelData = await addMemberGroup({
+      channelId,
+      memberIds: selectedUsers.map((user) => user.user_id)
+    });
+
     try {
-      const result = await channel.removeMembers([selectedUser.user_id || selectedUser.userId]);
-      const updateParticipant = newParticipant.filter(
-        (participant) => participant.user_id !== (selectedUser.user_id || selectedUser.userId)
+      const {channelName} = getChannelListInfo(
+        responseChannelData.data,
+        signedProfileId,
+        anonProfileId
       );
-      setNewParticipant(updateParticipant);
-      updateMemberName(result.members);
-      setNewParticipant(result.members);
-      setParticipants(result.members, groupPatchDispatch);
+      const channelList = await ChannelListSchema.getSchemaById(localDb, channelId);
+      channelList.name = channelName;
+      await channelList.save(localDb);
+
+      setSelectedChannel(channelList);
+
+      selectedUsers?.forEach(async (member) => {
+        const userMember = UserSchema.fromMemberWebsocketObject(
+          {
+            user: {
+              id: member?.user_id,
+              username: member?.username,
+              image: member?.profile_pic_path,
+              last_active: member?.last_active_at,
+              created_at: member?.created_at,
+              updated_at: member?.updated_at
+            },
+            banner: member?.is_banner
+          },
+          channelId
+        );
+        await userMember.saveOrUpdateIfExists(localDb);
+      });
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'error');
-      }
+      console.log('error on memberSchema');
+      console.log(JSON.stringify(e));
+      console.log(e);
     }
+
+    refresh('channelList');
+    refresh('chat');
+    refresh('channelInfo');
+
+    setTimeout(() => {
+      navigation.navigate('SignedChatScreen');
+    }, 500);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onAddMember` function has been added to handle adding members to the group. Similar to the previous comment, consider improving error handling and logging.
```diff
- console.log('error on memberSchema');
+ console.error('Error while updating member schema in onAddMember:', e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->